### PR TITLE
Divyanshu | Test | Provide the pagination at the bottom of the page after user scroll the page

### PR DESCRIPTION
**Pull request by dvCodeWorld**
https://app.clickup.com/t/86ctvdffp
https://app.clickup.com/t/86ctvdfpj

### CODE REVIEW

File: apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html [hunk 1]
```diff
@@ -226,10 +226,9 @@ <h2 class="mr-2 head-report" *ngIf="groupStockReport">
         <div class="mobile-card-wrapper">
             <table
                 class="table basic table-border inventory-stock-table on-mobile-view table-responsive"
-                [ngClass]="{
-                    'mr-b100':
-                        groupStockReport && groupStockReport.stockReport && groupStockReport.stockReport.length > 6
-                }"
+                [ngClass]="{ 'mr-b100': groupStockReport && groupStockReport.stockReport && groupStockReport.stockReport.length > 6 }
+                            ||
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html]: Check this condition is it required?

File: apps/web-giddh/src/app/inventory/components/stock-report-component/inventory.stockreport.component.html [hunk 2]
```diff
@@ -86,7 +86,7 @@ <h2 class="head-report" *ngIf="stockReport && stockReport.stockUnit">
                 class="table basic on-mobile-view table-responsive"
                 [ngClass]="{
                     'mr-b100': stockReport && stockReport.transactions && stockReport.transactions.length > 6
-                }"
+                } || { 'mb-5': stockReport && stockReport.totalPages > 1 }"
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/inventory/components/stock-report-component/inventory.stockreport.component.html]: Check this condition also

File: apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.html [hunk 3]
```diff
@@ -183,14 +183,16 @@
                 <tr mat-row *matRowDef="let row; columns: displayedColumns"></tr>
             </table>
         </section>
-        <div class="pagination-wrapper mt-4" [hidden]="totalItems <= paginationLimit">
+        <div class="pagination-wrapper mt-4" *ngIf="(totalPages > 1) && !isReportLoading">
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.html]:
```suggestion
        <div class="pagination-wrapper mt-4" *ngIf="(totalPages > 1) && !isReportLoading">
       Check this condition this is correct.
```

File: apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.ts [hunk 4]
```diff
@@ -144,6 +148,9 @@ export class ListManufacturingComponent implements OnInit {
         this.currentOrganizationType = this.generalService.currentOrganizationType;
         this.getWarehouses();
 
+        this.manufacturingSearchRequest.count = this.paginationLimit;
+        this.manufacturingSearchRequest.page = this.currentPage;
```
Comment (dilpreetsinghofficial) [inline apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.ts:152]: Confirmed with Ravinder sir

**Review by dilpreetsinghofficial — CHANGES_REQUESTED** [review]

File: apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html (hunk 1 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html]: Not Required, Removed

File: apps/web-giddh/src/app/inventory/components/stock-report-component/inventory.stockreport.component.html (hunk 2 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/inventory/components/stock-report-component/inventory.stockreport.component.html]: Not Required, Removed

File: apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.ts (hunk 4 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.ts:152]: Confirmed with ravinder sir

File: apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.html (hunk 3 above)
Comment (dvCodeWorld) [reply apps/web-giddh/src/app/new-inventory/component/manufacturing/list-manufacturing/list-manufacturing.component.html]: Done

**Review by dilpreetsinghofficial — APPROVED** [review]

**Review by ravinderwalkover — APPROVED** [review]

**Pull request merged**